core/src/main/scala/com/github/johnynek/paiges/Doc.scala

```diff
@@ -619,6 +619,13 @@ object Doc {
     }
 
   /**
+   * Build a document from a single character.
+   */
+  def char(c: Char): Doc =
+    if (32 <= c && c <= 126) charTable(c.toInt)
```

Comment:
and if this doesn't fail CI, we should add a test for `char(c) == text(c.toString)` I think, and also `char(c).render(0) == c.toString`